portfolio.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Georgia Adam</title>
+  <link rel="shortcut icon" href="/assets/forest.png">
+  <link rel="stylesheet" href="/styles/style.css">
+  <link rel="stylesheet" href="/styles/portfolio.css">
+
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li><a href="/index.html"> Home </a></li>
+      <li><a href="/portfolio.html"> Portfolio </a></li>
+      <li><a href="/about.html"> About </a></li>
+    </ul>
+  </nav>
+  
+  <header>
+    <h1>Georgia Adam</h1>
+    <h2>Software Engineering Student @ Ada Developers Academy</h2>
+  </header>
+
+  <section id="blurb">
+    <h3>
+      Please find a selection of projects I have worked on below.
+    </h3>
+  </section>
+
+  <main class="flex_container">
+    <div><a href="https://github.com/georgia-adam/task-list-api"><h2>Task List</h2></a><h3>Flask API</h3></div>
+    <div><a href="https://github.com/georgia-adam/book-review-site"><h2>Book Review Site</h2></a><h3>HTML/CSS</h3></div>
+    <div><a href="https://github.com/georgia-adam/swap-meet"><h2>Swap Meet</h2></a><h3>Python OOP</h3></div>
+    <div><a href="https://github.com/georgia-adam/viewing-party"><h2>Viewing Party</h2></a><h3>Python</h3></div>
+  </main>
+
+  <section class="contact">
+    <h3>Contact</h3>
+    <ul>
+      <li><a href="mailto:georgiaeadam@gmail.com"><img alt="Email" src="assets/square.png"/></a></li>
+      <li><a href="https://www.linkedin.com/in/georgiaeadam/"><img alt="LinkedIn" src="assets/linkedin.png"/></a></li>
+    </ul>
+  </section>
+
+  <footer>
+    <h4>&copy; 2022 Georgia Adam <br>
+    Photo by <a href="https://unsplash.com/@aewild?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Amanda Frank</a> on <a href="https://unsplash.com/@aewild?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a> / 
+    <a href="https://www.flaticon.com/free-icons/tree" title="tree icons">Tree icons created by Freepik - Flaticon</a> / 
+    <a href="https://www.flaticon.com/free-icons/at" title="at icons">At icons created by Freepik - Flaticon</a> / 
+    <a href="https://www.flaticon.com/free-icons/linkedin" title="linkedin icons">Linkedin icons created by Freepik - Flaticon</a>
+    </h4>
```

Comment:
The h1-h6 tags are intended to represent the heading of a section. Here the main content of the footer is embedded in the `h4` element. This can make it harder for screen reader users to understand and navigate the content. I would recommend using one or more <p> elements here and styling as needed to get the boldness and such you want from the font.